Cargo.lock

```diff
@@ -2297,6 +2298,11 @@ name = "version_check"
 version = "0.1.4"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 
+[[package]]
+name = "vfs"
```

Comment:
I cannot be sure this module is enough to use in production. Why do you choose it?